docs/notebook-demos/reading-in-sentinel-data.ipynb

```diff
@@ -0,0 +1,244 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Reading in Sentinel-2 Images\n",
+    "\n",
+    "Sentinel-2 is an observation mission developed by the European Space Agency to monitor the surface of the Earth [official website](http://www.esa.int/Our_Activities/Observing_the_Earth/Copernicus/Sentinel-2).\n",
+    "Sets of images are taken of the surface where each image corresponds to a specific wavelength. These images can provide useful data for a wide variety of industries, however, the format they are stored can prove difficult to work ith. This being, `JPEG 2000` (file extension `.jp2`), an image compression\n",
```

Comment:
...the format they are stored **in** ... can prove difficult to work **with**.  This being JPEG 2000 (no comma)